packages/query/src/yargs.ts

```diff
@@ -33,6 +33,11 @@ export function getYargsOption() {
       default: 'info',
       choices: ['fatal', 'error', 'warn', 'info', 'debug', 'trace', 'silent'],
     },
+    meta: {
```

Comment:
`--meta` seems not very clear to me.
```suggestion
    indexer: {
```
and pass the root endpoint of indexer. You might need /health of indexer as well.